completions/convert

```diff
@@ -49,7 +49,11 @@ _comp_cmd_convert__common_options()
             return
             ;;
         -format)
-            _comp_compgen_split -- "$(convert -list format | _comp_awk \
+            # FIXME: We should probably accept the actual command being used
+            # here and run that, instead of calling `magick` or `convert`
+            # explicitly.
+            # `magick` is the new way to call the command in image magick 7
+            _comp_compgen_split -- "$(magick -list format 2>/dev/null || convert -list format 2>/dev/null | _comp_awk \
```

Comment:
```suggestion
            _comp_compgen_split -- "$({ magick -list format 2>/dev/null || convert -list format 2>/dev/null; } | _comp_awk \
```

`a || b | c` would be treated as `a || { b | c; }`. In addition, formatting by `shfmt` might be needed.

Reply:
Did the first suggestion. Regarding the second, I accidentally left over the `2>/dev/null` on the `convert` from a different thing I checked, removed it.